R/pseudo_srcref.R

```diff
@@ -26,5 +26,15 @@ as.double.with_pseudo_srcref <- function(x, ...) {
 
 #' @export
 format.with_pseudo_srcref <- function(x, ...) {
+  if (inherits(x[[1]], "standardGeneric")) {
+    return(format_standardGeneric_call(x))
+  }
+
   format(unclass(x), ...)
 }
```

Comment:
This change isn't strictly necessary to resolve this bug, but handles a downstream issue of not being able to render test srcrefs if they include one of these generic calls. A necessary change to allow our vignettes to continue building.